presto-docs/src/main/sphinx/release/release-0.207.rst

```diff
@@ -0,0 +1,71 @@
+=============
+Release 0.207
+=============
+
+General Changes
+---------------
+
+* Fix a planning issue for queries where correlated references were used in ``VALUES``.
+* Remove support for legacy ``JOIN ... USING`` behavior.
+* Change behavior for unnesting an array of ``row`` type to produce multiple columns.
+* Deprecate the ``reorder_joins`` session property and the ``reorder-joins`` configuration property.
+  Replace with the ``join_reordering_strategy`` session property and the
+  ``optimizer.join-reordering-strategy`` configuration propery. Setting this new property to ``NONE``
+  maintains the order of the joins as written and is equivalent to ``reorder-joins=false``.
+  ``ELIMINATE_CROSS_JOINS`` will eliminate any unnecessary cross joins from the plan and is
+  equivalent to ``reorder-joins=true``. Setting it to ``COST_BASED`` will use the new cost-based
+  optimizer to select the best join order. To simplify migration, setting the ``reorder_joins``
+  session property overrides the new session and configuration properties.
+* Deprecate the ``distributed_joins`` session property and the ``distributed-joins-enabled`` configuration
```

Comment:
```
* Deprecate the ``distributed_joins`` session property and the
  ``distributed-joins-enabled`` configuration property. They are replaced by the
  ``join_distribution_type`` session property and the ``join-distribution-type``
  configuration property. ``REPARTITIONED`` turns on hash partitioned joins and
  is equivalent to ``distributed_joins-enabled=true``. ``BROADCAST`` changes the
  join strategy to broadcast and is equivalent to ``distributed_joins-enabled=false``.
  ``AUTOMATIC`` will use the new cost-based optimizer to select the best join
  strategy. If no statistics are available, ``AUTOMATIC`` is the same as
  ``REPARTITIONED``. To simplify migration, setting the ``distributed_joins``
  session property overrides the new session and configuration properties.

Reply:
Why do we use ``AUTOMATIC`` for this one and ``COST_BASED`` for the other?

Reply:
@rschlussel

Reply:
No good reason